client/src/views/Models/components/Graphs/GlobalGraph.vue

```diff
@@ -38,11 +38,16 @@
       this.refresh();
     }
 
+    @Watch('layout')
+    layoutChanged (): void {
+      this.refresh();
+    }
+
     mounted (): void {
        this.renderer = new EpiRenderer({
         el: this.$refs.graph,
-        adapter: new DagreAdapter(DEFAULT_RENDERING_OPTIONS),
-        renderMode: 'delta',
+        adapter: new ELKAdapter(Object.assign({}, DEFAULT_RENDERING_OPTIONS, { layout: layered })),
```

Comment:
If `layered` is only used by `ELKAdapter` why not importing it inside `ELKAdapter` and do the `Object.assign()` outside of this compoment.

Reply:
Done.